src/app/components/common/node-editor/custom-node/custom-node.component.ts

```diff
@@ -83,32 +101,56 @@ export class CustomNodeComponent implements OnChanges {
 
   ngOnChanges(): void {
     this.nodeId = this.data.id;
-    if (this.data.label === 'South') {
+    if (this.data.label === 'South' || this.data.label === 'North') {
       if (this.source !== '') {
+        this.isServiceNode = true;
         this.elRef.nativeElement.style.borderColor = "#B6D7A8";
         interval(POLLING_INTERVAL)
           .pipe(takeWhile(() => this.isAlive), takeUntil(this.destroy$)) // only fires when component is alive
           .subscribe(() => {
-            this.getSouthboundServices();
+            if (this.from == 'north') {
+              this.getNorthboundTasks();
```

Comment:
Should we replace Task with Instance? Given North Instance can be a service or task.